apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,367 @@
+    public class BigtableClientSnippets
+    {
+        private readonly BigtableClientSnippetsFixture _fixture;
+
+        public BigtableClientSnippets(BigtableClientSnippetsFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public async Task CheckAndMutateRowAsync()
+        {
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                Mutations.SetCell("Address", "Country", "USA"));
+
+            // Snippet: CheckAndMutateRowAsync(TableName,BigtableByteString,RowFilter,*)
+            CheckAndMutateRowResponse response = await client.CheckAndMutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                // If the latest value in the "Address:Country" column is "USA"...
+                RowFilters.Chain(
+                    RowFilters.FamilyNameRegex("Address"),
+                    RowFilters.ColumnQualifierRegex("Country"),
+                    RowFilters.CellsPerColumnLimit(1),
+                    RowFilters.ValueRegex("USA")),
+                // ...remove all previous Level 1 scores and set the value to 0.
+                Mutations.DeleteFromColumn("Score", "Level 1"),
+                Mutations.SetCell("Score", "Level 1", 0, new BigtableVersion(1)));
+
+            if (response.PredicateMatched)
+            {
+                Console.WriteLine("The level 1 scores have been reset");
+            }
+            else
+            {
+                Console.WriteLine("The level 1 scores were not modified.");
+            }
+            // End snippet
+
+            Assert.True(response.PredicateMatched);
+        }
+
+        [Fact]
+        public async Task MutateRowAsync()
+        {
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                new byte[] { 0xAB, 0xCD, 0x12, 0x34 },
+                Mutations.SetCell("Address", "Country", "USA"),
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: MutateRowAsync(TableName,BigtableByteString,*)
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                // From the row with key '0xABCD1234'...
+                new byte[] { 0xAB, 0xCD, 0x12, 0x34 },
+                // delete all cells from the column family "Address",
+                Mutations.DeleteFromFamily("Address"),
```

Comment:
Neither of these methods return anything saying how many cells/rows were deleted, I assume?

Reply:
That's correct. MutateRowResponse contains no information (see [here](https://github.com/googleapis/googleapis/blob/546d3afd75187e65229a2670eb440e84a05f5d9d/google/bigtable/v2/bigtable.proto#L245))